pandas/core/groupby/groupby.py

```diff
@@ -1323,7 +1332,17 @@ def sem(self, ddof: int = 1):
         Series or DataFrame
             Standard error of the mean of values within each group.
         """
-        return self.std(ddof=ddof) / np.sqrt(self.count())
+        result = self.std(ddof=ddof)
+        if result.ndim == 1:
+            result /= np.sqrt(self.count())
+        else:
+            cols = result.columns.get_indexer_for(
+                result.columns.difference(self.exclusions).unique()
+            )
+            result.iloc[:, cols] = (
```

Comment:
you can use /= as well

Reply:
Unfortunately not. using /= follow the same path within iLocIndexer._setitem_with_indexer which is broken when there are duplicate labels. Only by using .values on the numerator and denominator are the labels stripped, and then it goes through okay.